.github/workflows/release-check.yml

```diff
@@ -0,0 +1,44 @@
+name: Release Checker
+
+on:
+  pull_request_target:
+    paths: [
+      "dispatch_examples/greeter/Cargo.toml",
+      "frc42_dispatch/Cargo.toml",
+      "frc42_dispatch/hasher/Cargo.toml",
+      "dispatch_examples/greeter/Cargo.toml",
+      "frc42_dispatch/Cargo.toml",
+      "frc42_dispatch/hasher/Cargo.toml",
+      "frc42_dispatch/macros/Cargo.toml",
+      "frc46_token/Cargo.toml",
+      "frc53_nft/Cargo.toml",
+      "fvm_actor_utils/Cargo.toml",
+      "fvm_dispatch_tools/Cargo.toml"
+    ]
+    types: [ opened, synchronize, reopened, labeled, unlabeled ]
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release-check:
+    uses: ipdxco/unified-github-workflows/.github/workflows/release-check.yml@v1.0
+    with:
+      sources: |
```

Comment:
Can we use yaml references/anchors to deduplicate this list?

Reply:
Alternatively, at least for the `paths:` part, we can probably just glob on `**/Cargo.toml`. We'll get some false positives, but that'll be fine.

Reply:
Unfortunately, no advanced yaml features for GitHub Actions. We can definitely do `**/Cargo.toml`. I like this idea - will add to this branch.